hazelcast-jet-core/src/main/java/com/hazelcast/jet/windowing/WindowOperation.java

```diff
@@ -83,13 +88,22 @@
      * #combineAccumulatorsF() combine} so that {@code deduct(combine(acc, x),
      * x)} returns an accumulator in the same state as {@code acc} was before
      * the operation.
+     * <p>
+     * <b>Note:</b> this operation is optional and can return {@code null}. If
```

Comment:
It's not the operation that can return `null`. Also the text should consistently use either "operation" or "function" to refer to the object. Since we use "operation" for the entire type (a tuple of functions), "function" would be better.